docs/api/en/objects/InstancedMesh.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<base href="../../../" />
+		<script src="list.js"></script>
+		<script src="page.js"></script>
+		<link type="text/css" rel="stylesheet" href="page.css" />
+	</head>
+	<body>
+		[page:Mesh] &rarr;
+
+		<h1>[name]</h1>
+
+		<p class="desc">
+		A special version of [page:Mesh] with instanced rendering support. Use [name] if you have to render a large amount of
+		objects with the same geometry and material but with different world transformations. The usage of [name] will help you
+		to reduce the amount of draw calls and thus improve the overall rendering performance in your application.
+		</p>
+
+		<h3>Example</h3>
+
+		<p>
+			[example:webgl_instancing_suzanne WebGL / instancing / suzanne]<br />
+		</p>
+
+		<h2>Constructor</h2>
+		<h3>[name]( [param:BufferGeometry geometry], [param:Material material], [param:Integer count] )</h3>
+		<p>
+		[page:Geometry geometry] - an instance of [page:BufferGeometry].<br />
+		[page:Material material] - an instance of [page:Material]. Default is a new [page:MeshBasicMaterial].<br />
+		[page:Integer count] - the amount of instances.<br />
+		</p>
+
+		<h2>Properties</h2>
+		<p>See the base [page:Mesh] class for common properties.</p>
+
+		<h3>[property:Integer count]</h3>
+		<p>
+			The amount of instances. The *count* value passed into the constructor represents the maximum amount of
+			instances of this mesh. You can change the amount of instances at runtime to an integer value
+			in the range [0, count].
+		</p>
+		<p>
+			If you need more instances than the original count value, you have to create a new [name].
+		</p>
+
+		<h3>[property:Boolean isInstancedMesh]</h3>
```

Comment:
Not anything to fix in this PR, but maybe we should have a "readonly" tag for this and other similar cases in the API. 🤔

Reply:
I don't think we should document them actually...

Reply:
Then let me remove it^^